apps.awesim.org/apps/myjobs/templates/Basic_AMBER_Job_Pitzer-Exp/basic_amber.sh

```diff
@@ -0,0 +1,31 @@
+#PBS -N ondemand/sys/myjobs/basic_amber
+#PBS -j oe
+#PBS -l walltime=00:10:00
+#PBS -l nodes=2:ppn=40
```

Comment:
I'm pretty sure all these directives have to change to their `#SBATCH` equivalent.

Reply:
FWIW they can stay and should work but I'd very much like to see OSC using native SLURM directives and not rely on the PBS compatibility layer provided by SLURM.

Reply:
Working on converting them all now.

Reply:
Just drop `-j oe`, that is default with SLURM.  For walltime use --time. For nodes=2 use --nodes=2 and ppn=40 is --ntasks-per-node=40.

```
#SBATCH --time=00:10:00
#SBATCH --nodes=2
#SBATCH --ntasks-per-node=40
```

Also be aware Pitzer expansion is 48 core nodes so maybe rather than --ntasks-per-node just do --exclusive.

If we have any references to `mpiexec` or `mpirun` they will need to be converted to `srun`.

Reply:
If you do `--exclusive` then for srun you do something like `srun --ntasks-per-node=$SLURM_CPUS_ON_NODE` to use all cores on the node.